src/resources/spi_linux.cc

```diff
@@ -27,7 +27,7 @@
 
 namespace toit {
 
-// Defined in primitive_file_posix.cc.
+// Defined in primitive_file_posix_and_esp32.cc.
```

Comment:
Not super excited about the `and` in the filename, but I can't think of something better.

Reply:
We would probably use `shared` if it was for all platforms. Maybe there is a term we can introduce that covers more here.

Alternatively, we can think about splitting it out (have a dedicated `primitive_file_esp32`) and share code in a more granular way. I suggest we wait with that until we bring up support for another platform like Zephyr.

Reply:
primitive_file_non_windows.cc
?

Reply:
Or primitive_file_non_win.cc to fit with other windows-related source files.